### PR TITLE
Fix fast forward/rewind not working on Fire Stick

### LINKED ISSUE #351: Fast forward and rewind buttons no longer work with new player overlay
It seems the fast forward and rewind buttons on Fire TV remotes no longer work after the player overlay redesign.

### PR DESCRIPTION
**Pull request by MrLemur**
**Changes**
This adds in some new logic to intercept the fast forward and rewind
keys presses if the popup panel is not in view.

If the media playing is not live TV, it will skip forwards or backwards
the appropriate amount of seconds. It will show the overlay as expected]
and then initiate the fadeout timer.

There is also a filter that has been added to only intercept the `ACTION_DOWN` of the keypress. Before, this would cause a fast forward/rewind to be done twice, effectively doubling the intended seek time.

**Issues**
Fixes #351 

I would appreciate any testing on other Fire Sticks, this has only been tested on a Fire Stick 4K.

### CODE REVIEW

**Comment by MrLemur** [comment]
@thornbill @nielsvanvelzen I understand you might have some devices to test with?

File: app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java [hunk 1]
```diff
@@ -432,6 +432,7 @@ public void onItemClicked(Presenter.ViewHolder itemViewHolder, Object item,
     private View.OnKeyListener keyListener = new View.OnKeyListener() {
         @Override
         public boolean onKey(View v, int keyCode, KeyEvent event) {
+            if (event.getAction() == KeyEvent.ACTION_DOWN) {
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java:435]: The formatting inside this if statement is wrong. Please indent it or change the code to return false when the action is not ACTION_DOWN

File: app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java [hunk 2]
```diff
@@ -501,6 +502,23 @@ public boolean onKey(View v, int keyCode, KeyEvent event) {
...
+
+                    if (keyCode == KeyEvent.KEYCODE_MEDIA_REWIND || keyCode == KeyEvent.KEYCODE_BUTTON_L1 || keyCode == KeyEvent.KEYCODE_BUTTON_L2) {
+                        mPlaybackController.skip(-11000);
+                        if (!mIsVisible) show();
+                        setFadingEnabled(true);
+                        return true;
+                    }
+                }
+
                 if (!mIsVisible) {
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java:522]: I expect everything inside this statement to probably trigger twice on non-firetv devices

File: app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java (hunk 1 above)
Comment (MrLemur) [reply app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java:436]: What are the current indent preferences for java files?

File: app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java (hunk 2 above)
Comment (MrLemur) [reply app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java]: Will be good to get a test to see, and if so I can add in the fire TV check.

File: app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java (hunk 1 above)
Comment (nielsvanvelzen) [reply app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java:436]: It's 4 spaces currently

File: app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java (hunk 2 above)
Comment (MrLemur) [reply app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java]: Testing in Android TV emulator in AVD has the expected behaviour using the emulated d-pad fast forward and rewind controls.

File: app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java (hunk 1 above)
Comment (MrLemur) [reply app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java:436]: Should be fixed with the most recent commit

File: app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java (hunk 2 above)
Comment (nielsvanvelzen) [reply app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java]: I did some testing and it, indeed, works as expected! 🎉

**Pull request merged**